bindings/go/Makefile.am

```diff
@@ -11,8 +11,10 @@ CGOCFLAGS=-I$(abs_top_builddir)/src -I$(top_srcdir)/src
 CGOLDFLAGS=-L$(abs_top_builddir)/src/ucp/$(objdir) -lucp -L$(abs_top_builddir)/src/ucs/$(objdir) -lucs
 UCX_SOPATH=$(abs_top_builddir)/src/ucp/$(objdir):$(abs_top_builddir)/src/ucs/$(objdir):$(abs_top_builddir)/src/ucm/$(objdir):$(abs_top_builddir)/src/uct/$(objdir)
 GOTMPDIR=$(GOOBJDIR)/tmp
+GOCACHE=$(GOTMPDIR)
 
 export GOTMPDIR
+export GOCACHE
```

Comment:
is it really needed in this PR?
maybe submit as a separate PR?

Reply:
Will submit a separate PR and delete related change from this PR.

Reply:
https://github.com/openucx/ucx/pull/7691 is for GOCACHE.